### PR TITLE
[FSDE-939] Fix relative routes for smoke tests

### PR DESCRIPTION
**Pull request by dani-cl-madrid**

### CODE REVIEW

File: config/layouts/test.pug [hunk 1]
```diff
@@ -8,12 +8,12 @@ html(lang="en")
     meta(name="description", content="Chi Tests")
     meta(name="viewport", content="width=device-width, initial-scale=1")
 
-    link(rel="stylesheet", media="all", href="/chi.css")
-    link(rel="stylesheet", media="all", href="/tests/test.css")
+    link(rel="stylesheet", media="all", href="../../chi.css")
```
Comment (janibolkvadze) [inline config/layouts/test.pug:11]:
```suggestion
    link(rel="stylesheet", media="all", href="../../chi.css")
    script(src=`../../js/chi.js`)
```

**Review by janibolkvadze — CHANGES_REQUESTED** [review]

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-102/1/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://09968586dcc1f5a414a98ed52fb67dd9e1b44adf-chi-pr.enterprise.ctl.io

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-102/2/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://7fa9051f20f43f1675d0f367b699ec567526fb91-chi-pr.enterprise.ctl.io

**Review by julianatienza — APPROVED** [review]

File: config/layouts/test.pug [hunk 2]
```diff
@@ -8,12 +8,13 @@ html(lang="en")
...
+    script(src=`../../js/chi.js`)
```
Comment (janibolkvadze) [inline config/layouts/test.pug:12]:
```suggestion
```

**Review by janibolkvadze — CHANGES_REQUESTED** [review]

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-102/3/. :x:

**Review by janibolkvadze — APPROVED** [review]

**Review by jllr — APPROVED** [review]

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-102/4/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://da26be4d407fcf6dc447b4b357e47176c4ab983b-chi-pr.enterprise.ctl.io

**Pull request merged**